src/signature_pad.ts

```diff
@@ -563,9 +563,7 @@ export default class SignaturePad {
       '<svg' +
       ' xmlns="http://www.w3.org/2000/svg"' +
       ' xmlns:xlink="http://www.w3.org/1999/xlink"' +
-      ` viewBox="${minX} ${minY} ${maxX} ${maxY}"` +
-      ` width="${maxX}"` +
-      ` height="${maxY}"` +
```

Comment:
@mesqueeb Why did you remove width and height?

Reply:
Sorry!
I never include it on my SVGs, viewBox should be enough.